main.css

```diff
@@ -37,45 +39,54 @@ html, body
 #header
 {
     position: fixed;
-    height: 30px;
+    height: 35px;
```

Comment:
There's some overlap now with the sidebar. Also I think we can afford for it to be a _little_ bigger, maybe `45px` instead?